vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -25,196 +25,9 @@
-    - name: SHELL
-      value: /bin/bash
-
-- name: schedule the job to run every day at 2 AM
-  become: yes
-  become_user: pentaho-pdi
-  cron:
-    backup: yes
-    name: "populate FOIA data warehouse tables"
-    job: "{{ root_folder }}/app/pentaho-pdi/data-integration/kitchen.sh -file://{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/foia-dw1.kjb"
-    month: "*"
-    weekday: "*"
-    day: "*"
-    hour: "2"
-    minute: "0"
-    disabled: no
-    state: present
-
-###################################################
-# Create the JDBC and the Mondrian data sources 
-###################################################
-
-- name: list the data sources that already exist
-  become: yes
-  become_user: pentaho
-  command: 
-    cmd: 'wget --no-check-certificate -O {{ root_folder }}/install/pentaho/datasources-list.json --user={{ ldap_user_prefix }}arkcase-admin --password "{{ arkcase_admin_password }}" https://{{ internal_host }}:2002/pentaho/plugin/data-access/api/datasource/jdbc/connection'
-    warn: no
-  changed_when: false
-
-- name: read the list of data sources
-  become: yes
-  become_user: pentaho
-  command: cat {{ root_folder }}/install/pentaho/datasources-list.json
-  register: data_source_list
-  changed_when: false
-
-- name: file to post to Pentaho to create the data source
-  become: yes
-  become_user: pentaho
-  template:
-    src: put_connection.json
-    dest: "{{ root_folder }}/install/pentaho/put_connection.json"
-  when: not '"MariaDB Local"' in data_source_list.stdout
-
-# the ansible uri module causes a timeout for some reason but wget works fine
-- name: create the ArkCase JDBC data source for analytical reports
-  command: 
-    cmd: 'wget --no-check-certificate --user={{ ldap_user_prefix }}arkcase-admin --password "{{ arkcase_admin_password }}" --post-file {{ root_folder }}/install/pentaho/put_connection.json --header="Content-Type: application/json" https://{{ internal_host }}:2002/pentaho/plugin/data-access/api/connection/add'
-    warn: no
-  register: create_datasource_rest_call
-  when: not '"MariaDB Local"' in data_source_list.stdout
-
-- name: upload the Mondrian schema
-  become: yes
-  become_user: pentaho
-  command:
-    cmd: 'curl -k -v -H "Content-Type: multipart/form-data" -X PUT -F uploadInput=@{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/mondrian_schema/{{ foia_mondrian_schema_file }} -F overwrite=true -F xmlaEnabledFlag=false -F parameters="Datasource=MariaDB Local" -u {{ ldap_user_prefix }}arkcase-admin:{{ arkcase_admin_password }} https://{{ internal_host }}:2002/pentaho/plugin/data-access/api/datasource/analysis/catalog/{{ foia_mondrian_schema_file | replace(".xml", "") }}'
-    warn: no
-
-#########################################################
-# Deploy the reports 
-#########################################################
+- set_fact:
```

Comment:
I'm going to guess you didn't mean to include these foia-analytical-report changes?

Reply:
That's what you get when you type git commit -am 🥇